pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -266,15 +268,15 @@ func (a *Bootstrap) addSystemdUnits(uri string, templateData *bootstrapTemplateD
 					return err
 				}
 
-				dropins = append(dropins, igntypes.SystemdDropin {
+				dropins = append(dropins, igntypes.SystemdDropin{
```

Comment:
nit: These 0ff89ad20 gofmt fixes should be squashed down into 0733352 instead.

Reply:
Thanks. I just noticed that as well.